mmpose/datasets/datasets/__init__.py

```diff
@@ -1,13 +1,14 @@
 from .bottom_up import BottomUpCocoDataset
+from .mesh import MeshAdvDataset, MeshH36MDataset, MeshMixDataset, MoshDataset
 from .top_down import (TopDownAicDataset, TopDownCocoDataset,
                        TopDownCocoWholeBodyDataset, TopDownFreiHandDataset,
                        TopDownMpiiDataset, TopDownMpiiTrbDataset,
-                       TopDownOCHumanDataset, TopDownOneHand10KDataset,
-                       TopDownPanopticDataset)
+                       TopDownOCHumanDataset, TopDownOneHand10KDataset)
```

Comment:
TopDownPanopticDataset was accidently removed

Reply:
I will fix this bug.